src/Sulu/Bundle/WebsiteBundle/DataCollector/SuluCollector.php

```diff
@@ -1,157 +0,0 @@
-use Sulu\Component\Webspace\Portal;
-use Sulu\Component\Webspace\Webspace;
-use Sulu\Content\Domain\Model\DimensionContentInterface;
-use Sulu\Content\Domain\Model\ShadowInterface;
-use Sulu\Content\Domain\Model\TemplateInterface;
-use Sulu\Content\Domain\Model\WorkflowInterface;
-use Sulu\Page\Domain\Model\PageDimensionContentInterface;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\HttpKernel\DataCollector\DataCollector;
-
-class SuluCollector extends DataCollector
-{
-    public function __construct(
-        private string $kernelEnvironment = 'dev'
-    ) {
-    }
-
-    public function data(string|int $key): mixed
-    {
-        return $this->data[$key] ?? null;
-    }
-
-    public function collect(Request $request, Response $response, ?\Throwable $exception = null): void
-    {
-        if (!$request->attributes->has('_sulu')) {
-            return;
-        }
-
-        /** @var RequestAttributes $requestAttributes */
-        $requestAttributes = $request->attributes->get('_sulu');
-
-        /** @var ?Webspace $webspace */
-        $webspace = $requestAttributes->getAttribute('webspace');
-        /** @var ?Portal $portal */
-        $portal = $requestAttributes->getAttribute('portal');
-        $segment = $requestAttributes->getAttribute('segment');
-
-        $this->data['match_type'] = $requestAttributes->getAttribute('matchType');
-        $this->data['redirect'] = $requestAttributes->getAttribute('redirect');
-        $this->data['portal_url'] = $requestAttributes->getAttribute('portalUrl');
-        $this->data['segment'] = $requestAttributes->getAttribute('segment');
-
-        if ($webspace) {
-            $this->data['webspace'] = $webspace->toArray();
-            unset($this->data['webspace']['portals']);
-            $this->flattenLocalization($this->data['webspace']['localizations']);
-        }
-
-        if ($portal) {
-            $this->data['portal'] = $portal->toArray();
-            $this->data['portal']['environments'] = \array_combine(
-                \array_column($this->data['portal']['environments'] ?? [], 'type'),
-                $this->data['portal']['environments'] ?? [],
-            );
-            $this->flattenLocalization($this->data['portal']['localizations']);
-            $this->data['environment'] = $portal->getEnvironment($this->kernelEnvironment);
-        }
-
-        if ($segment) {
-            $this->data['segment'] = $segment->toArray();
-        }
-
-        $this->data['localization'] = $requestAttributes->getAttribute('localization');
-        $this->data['resource_locator'] = $requestAttributes->getAttribute('resourceLocator');
-        $this->data['resource_locator_prefix'] = $requestAttributes->getAttribute('resourceLocatorPrefix');
-
-        $structure = null;
-        if ($request->attributes->has('object')) {
-            $object = $request->attributes->get('object');
```

Comment:
Do you really want to remove everything? Somebody did already refactor most of the stuff in this service to work with the contentBundle. Removing this results in a loss of the contributions 🤔